README.md

```diff
@@ -1 +1,111 @@
-# Sea-Foods
\ No newline at end of file
+# Sea Foods
+
+## Table of Contents
+
+* [About the Project](#about-the-project)
+* [Built With](#built-with)
+* [Getting Started](#getting-started)
+* [API Interactions](#api-interactions)
+* [Contributing](#contributing)
+* [Author](#author)
+* [Acknowledgements](#acknowledgements)
+
+## About Project
+
+Sea Foods is a project that enlists dishes of sea life regarded as food. Users get to interact to items in the list by liking and commenting on them after viewing some more information on a particular dish on a popup modal.
+
+This project uses [TheMealDB](https://www.themealdb.com/api.php) API for populating the list of sea foods and the [Involvement](https://www.notion.so/Involvement-API-869e60b5ad104603aa6db59e08150270) API for adding likes and comments to any sea food item.
+
+![screenshot](./src/assets/img/screenshot.png)
+
+## [Live Demo](https://lameck1.github.io/Sea-Foods/)
+
+## [Video Demo Link](https://www.loom.com/share/1ef13d34279b43eab1f38b2957ad19df)
```

Comment:
- [ ] Great video presentation over there. However, you ought to have done it collaboratively. Despite seeing Olawale, he did not talk at all. Since you both worked on the project then each of you should present it to the client.